src/renderer/base/renderer.cpp

```diff
@@ -50,7 +51,7 @@ Renderer::Renderer(IRenderData* pData,
 // - <none>
 Renderer::~Renderer()
 {
-    _destructing = true;
+    // IRenderThread blocks until it has shut down.

```

Comment:
Worried about removing the `_destructing` member. Those things are usually workarounds for poor threading, and need to stay around until the threading is fixed :/